_articles/circuit-breaking.md

```diff
@@ -0,0 +1,148 @@
+specifics of your system as you set up circuit breaking.
+
+Circuit breaking is specified as part of a Cluster (a group of similar upstream
+hosts) definition by adding a `circuit_breakers` field. Clusters are returned
+from the Cluster Discovery Service (CDS), either in the bootstrap config, or a
+remote CDS server .
+
+## Circuit Breaker Configuration
+
+Here's what a simple circuit breaking configuration looks like:
+
+```yaml
+circuit_breakers:
+      thresholds:
+        - priority: DEFAULT
+          max_connections: 1000
+          max_requests: 1000
+        - priority: HIGH
+          max_connections: 2000
+          max_requests: 2000
+```
+
+In this example, there are a few fields that allow for a lot of service
+flexibility:
+
+ `thresholds` allows us to define priorities and limits for the type of traffic
+ that our service responds to.
+
+`priority` refers to how routes defined as `DEFAULT` or `HIGH` are treated by
+the circuit breaker. Using the settings above, we would want to set any
+requests that shouldn’t wait in a long queue to HIGH. For example: POST
+requests in a service where a user wants to make a purchase, or save their
+state.
+
+`max_connections` are the maximum number of connections that Envoy will make to
+our service clusters. The default for these is 1024, but in real-world
+instances we may drastically lower them.
+
+`max_requests` are the maximum number of parallel requests that Envoy makes to
+our service clusters. The default is also 1024.
+
+## Typical Circuit Breaker Policy
+
+Essentially all clusters will benefit from a simple circuit breaker at the
+network level. Because HTTP/1.1 and HTTP/2 have different connection behaviors
+(one connection per request vs. many requests per connection), clusters with
+different protocols will each use a different option:
+
+ - For **HTTP/1.1** connections, use `max_connections`.
+ - For **HTTP/2** connections, use `max_requests`.
+
+In both cases, these settings will ensure the circuit breaker is tripped when
+the majority of requests in the last 10 seconds have failed. A conservative
+starting point for a service that does 1,000 requests / second would be 8,000
+max connections / requests (80% of 10,000 requests).
+
+This will prevent catastrophic outages based on timeout failures, where a
+single service cascades everywhere. The most insidious failures happen not
+because a service is down, but because it’s hanging on to requests for tens of
+seconds. Generally, it’s better for one service to be completely down than for
+all services to be outside their SLO because they’re waiting for a timeout deep
+in the stack.
+
+## Advanced circuit breaking
+
+Now that you’ve seen a basic configuration and policy for circuit breaking,
+we’ll discuss more advanced circuit breaking practices. These advanced
+practices will add more resiliency to your infrastructure at the network level.
+
+###  Break on latency
```

Comment:
rm extra space before B

also, is this maybe "Break-on-latency" ? honestly not sure.